pyaedt/edb_core/edb_data/nets_data.py

```diff
@@ -171,3 +173,57 @@ def get_extended_net(self, resistor_below=10, inductor_below=1, capacitor_above=
         rlc_common = {i: v for i, v in comps_common.items() if v.type in ["Resistor", "Inductor", "Capacitor"]}
 
         return output_nets, rlc_common
+
+
+class EDBExtendedNetData(ExtendedNetDotNet):
+    """Manages EDB functionalities for a primitives.
+    It Inherits EDB Object properties.
+
+    Examples
+    --------
+    >>> from pyaedt import Edb
+    >>> edb = Edb(myedb, edbversion="2021.2")
+    >>> edb_extended_net = edb.nets.extended_nets["GND"]
+    >>> edb_extended_net.name # Class Property
+    """
+
+    def __init__(self, core_app, raw_extended_net=None):
+        self._app = core_app
+        self._core_components = core_app.components
+        self._core_primitive = core_app.modeler
+        self._core_nets = core_app.nets
+        ExtendedNetDotNet.__init__(self, self._app, raw_extended_net)
+
+    @pyaedt_function_handler
+    def add_nets(self, net_names: list[str]):
+        flag = True
+        for i in net_names:
+            flag = flag and self.add_net(i)
+        return flag
+
+
+class EDBDifferentialPairData(DifferentialPairDotNet):
+    """Manages EDB functionalities for a primitives.
+    It Inherits EDB Object properties.
```

Comment:
```suggestion
    """Manages EDB functionalities for a primitive.
    It inherits EDB object properties.
```
Changed to "a primitive" (singular object) but you might have meant "for primitives" (plural objects)?